xcdat/regridder/grid.py

```diff
@@ -376,16 +378,16 @@ def create_global_mean_grid(grid: xr.Dataset) -> xr.Dataset:
     xr.Dataset
         A dataset containing the global mean grid.
     """
-    lat = grid.cf["lat"]
+    lat = get_axis_coord(grid, "Y")
     lat_data = np.array([(lat[0] + lat[-1]) / 2.0])
 
-    lat_bnds = grid.cf.get_bounds("lat")
+    lat_bnds = grid.bounds.get_bounds("Y")
     lat_bnds = np.array([[lat_bnds[0, 0], lat_bnds[-1, 1]]])
 
-    lon = grid.cf["lon"]
+    lon = get_axis_coord(grid, "X")
     lon_data = np.array([(lon[0] + lon[-1]) / 2.0])
 
-    lon_bnds = grid.cf.get_bounds("lon")
+    lon_bnds = grid.bounds.get_bounds("X")
```

Comment:
I replace calls such as `grid.cf["lat"]` with `get_axis_coord()` because sometimes the name of the coordinate variable might not be "lat". 

`get_axis_coord()` tries to find the coord var using `axis`, `standard_name`, or dim name (e.g., "lat").